app/internal/page/attorney/register.go

```diff
@@ -141,22 +143,62 @@ func makeHandle(mux *http.ServeMux, store sesh.Store, errorHandler page.ErrorHan
 	}
 }
 
-func getProvidedDetails(appData page.AppData, lpa *page.Lpa) actor.Attorney {
+func getProvidedDetails(appData page.AppData, lpa *page.Lpa) actor.AttorneyProvidedDetails {
```

Comment:
I thought I'd just make these use the same pattern for now. Once we're done with the attorney journey and we refactor the data they'll disappear.